lib/ddtrace/contrib/redis/patcher.rb

```diff
@@ -60,6 +61,9 @@ def patch_redis_client
             def initialize(*args)
               pin = Datadog::Pin.new(SERVICE, app: 'redis', app_type: Datadog::Ext::AppTypes::DB)
               pin.onto(self)
+              if pin.tracer && pin.service && pin.app && pin.app_type
```

Comment:
Since the pin is created just before and we explicitly set those, no. Good catch.